docs/repo-guides/release-nnf-sw/readme.md

```diff
@@ -0,0 +1,161 @@
+---
+authors: Blake Devcich <blake.devcich@hpe.com>
+categories: release, repo
+---
+
+# Releasing NNF Software
+
+## NNF Software Overview
+
+The following repositories comprise the NNF Software and each have their own versions. There is a
+hierarchy of sorts, since `nnf-deploy` packages the individual components together using submodules.
+
+Each component under `nnf-deploy` needs to be released first, then `nnf-deploy` can be updated to
+point to those release versions, then `nnf-deploy` itself can be updated and released.
+
+The documentation repo (NearNodeFlash/NearNodeFlash.github.io) is released separately and is not
+part of `nnf-deploy`, but it should match the version number of `nnf-deploy`. Release this like the
+other components.
+
+- [NearNodeFlash/nnf-deploy](https://github.com/NearNodeFlash/nnf-deploy)
+    * [NearNodeFlash/nnf-dm](https://github.com/NearNodeFlash/nnf-dm)
+    * [NearNodeFlash/nnf-mfu](https://github.com/NearNodeFlash/nnf-mfu)
```

Comment:
A release of mfu requires an update to config/repositories.yaml.  The NNFMFU_VERSION variable must be updated.